google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -52,11 +52,12 @@ class Client
         ##
         # @private Creates a new Spanner Client instance.
         def initialize project, instance_id, database_id, min: 2, max: 10,
-                       keepalive: 1500
+                       keepalive: 1500, write_ratio: 0.5, fail: false
```

Comment:
I agree. Similar to how we use a `single_use` hash. I'll make the change and add documentation for it.

This method won't change, as it is marked private. I will only make this change on the public `Project#client` method.